ConsistencyManager/ConsistencyManager.swift

```diff
@@ -316,12 +317,11 @@ open class ConsistencyManager {
         dispatchTask { cancelled in
             let tuple = self.childrenAndListenersForModel(model)
             let optionalModelUpdates = CollectionHelpers.optionalValueDictionaryFromDictionary(tuple.modelUpdates)
-            self.updateListeners(
-                tuple.listeners,
-                withUpdatedModels: optionalModelUpdates,
-                context: context,
-                originalModel: model,
-                cancelled: cancelled)
+            self.updateListeners(tuple.listeners,
+                                 withUpdatedModels: optionalModelUpdates,
+                                 context: context,
+                                 originalModel: model,
+                                 cancelled: cancelled)
```

Comment:
Your style was correct previously. Look at the [swift style guide](https://github.com/linkedin/swift-style-guide)